### PR TITLE
jquery initial files

### PR DESCRIPTION
**Pull request by sailinnthu**
Please help to review jquery code.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,6 +3,10 @@
 <head>
 	<meta charset="UTF-8">
 	<title>Hello</title>
+	<script
+  src="https://code.jquery.com/jquery-3.2.1.min.js"
```
Comment (christopherthant) [inline index.html]: make sure your script is single line.

**Review by christopherthant — CHANGES_REQUESTED** [review]
check my comment and update

File: index.html (hunk 1 above)
Comment (sailinnthu) [reply index.html]: i had updated

**Comment by christopherthant** [comment]
look good.

**Comment by sailinnthu** [comment]
thanks

**Pull request merged**